src/stores/admin-signin.md

```diff
@@ -93,4 +93,21 @@ _Sign Out_
 
 The Sign In page displays a message that you are logged out. We recommend you sign out of the _Admin_ any time you leave your computer unattended.
 
+## Allow Multiple Admin Login
+
+The admin manages the orders, customers, products, shipping, and payments functionality. The default configuration is set to not allowing the admin account sharing for security purposes. Multiple admin users can manage the store faster and distribute the workload for better efficiency.
```

Comment:
This function actually determines if an account can be logged in at the same time from different devices. 

```suggestion
The Admin provides access to manage the orders, customers, products, shipping, and payments functionality. The default configuration is set to disallow multiple logins for an Admin user account as a security best practice. However, you can change this setting to allow Admin users to be logged in from multiple devices if that is needed to accommodate your business workflows.
```